src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -754,4 +754,20 @@ object messages {
            |${"square(ints: _*)          // res1: List[Int] = List(4, 9, 16)"}
            |""".stripMargin
   }
+
+  case class IllegalLiteral()(implicit ctx: Context) extends Message(28) {
+    val kind = "Syntax"
+    val msg = "illegal literal"
+    val explanation =
+      hl"""|Expected a valid literal.
+           |
+           |Available literals can be divided into the several groups:
+           | - Integer literals: 0, 21, 0xFFFFFFFF, -42L
+           | - Floating Point Literals: 0.0, 1e30f, 3.14159f, 1.0e-100, .1
+           | - Boolean Literals: true, false
+           | - Character Literals: 'a', '\u0041', '\\n'
```

Comment:
There shouldn't be an extra `\` in `'\\n'` right? You don't need to escape things (except `$`) in triple quoted strings.